app/decorators/flag_decorator.rb

```diff
@@ -0,0 +1,18 @@
+class FlagDecorator < ApplicationDecorator
+  delegate_all
+
+  def institution
+    # flag has trasncript id
+    # transcript belongs to a collection
+    # collection belongs to an institution
+
+    # rubocop:disable Style/TrailingCommaInArguments
+    institutions = Institution.joins(
```

Comment:
@sameera207 is it worth moving this into the Institution model?
Maybe something like `def self.institution_for_flag(flag_id)`

Reply:
@craighawkes , yeah, good point, I thought about it, but from what I see, flags will need the institutions not the other way around (at least at this point). Since it was a lengthy I had to start from Institution, not from the Flags. That is why I added the method to the flags decorator. But I will move the code to Institution if we start getting more requirments around this functionality